src/app/compiler.js

```diff
@@ -153,10 +155,11 @@ function Compiler (handleImportCall) {
       compilationFinished({error: 'Compiler not yet loaded.'})
     })
 
-    var newScript = document.createElement('script')
```

Comment:
A very similar piece of code is used here too: https://github.com/ethereum/browser-solidity/blob/master/src/app.js#L212

It seems to work for me on Chrome, though that doesn't mean it works everywhere.